Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1588,6 +1653,33 @@ namespace Remotion.BocAutoCompleteReferenceValue
         public unbind(): void {
             this.element && this.element.remove();
         }
+
+        private addLoadingAnnouncement() {
+            this.clearQueuedLoadingAnnouncement();
+            this.queuedLoadingAnnouncementTimeout = setTimeout(() => {
+                if (this.visible()) {
+                    this.announcementElement.innerText = this.options.loadingMoreMatchesMessage;
+                }
+            }, this.options.loadingMatchesMessageDelay);
+        }
+
+        private addLoadedAnnouncement() {
+            this.clearQueuedLoadingAnnouncement();
+            if (this.visible()) {
+                this.announcementElement.innerText = this.options.loadedMoreMatchesMessage;
+            }
+        }
+
+        private clearAnnouncements() {
+            this.announcementElement.innerText = "";
+        }
+
+        private clearQueuedLoadingAnnouncement() {
+            if (this.queuedLoadingAnnouncementTimeout !== null) {
+                clearTimeout(this.queuedLoadingAnnouncementTimeout);
+                this.queuedLoadingAnnouncementTimeout = null;
+            }
+        }
```

Comment:
I would like to do a quick rundown of this. announcementelement changes form UL to simple text?
Loading-message delay?